targets/liteos-stm32f4/source/main.c

```diff
@@ -0,0 +1,123 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "los_sys.h"
+#include "los_tick.h"
+#include "los_task.ph"
+#include "los_config.h"
+#include "los_bsp_led.h"
+#include "los_bsp_key.h"
+#include "los_bsp_uart.h"
+#include "los_bsp_adapter.h"
+
+#include "stm32f429i_discovery.h"
+#include <string.h>
+#include "jerryscript.h"
+#include "jerry_run.h"
+
+static UINT32 g_uwboadTaskID;
+LITE_OS_SEC_TEXT VOID LOS_BoadExampleTskfunc(VOID)
+{
+    while (1)
+    {
+	run();
+    }
+}
+void LOS_BoadExampleEntry(void)
+{
+    UINT32 uwRet;
+    TSK_INIT_PARAM_S stTaskInitParam;
+
+    (VOID)memset((void *)(&stTaskInitParam), 0, sizeof(TSK_INIT_PARAM_S));
+    stTaskInitParam.pfnTaskEntry = (TSK_ENTRY_FUNC)LOS_BoadExampleTskfunc;
+    stTaskInitParam.uwStackSize = LOSCFG_BASE_CORE_TSK_IDLE_STACK_SIZE;
+    stTaskInitParam.pcName = "BoardDemo";
+    stTaskInitParam.usTaskPrio = 10;
+    uwRet = LOS_TaskCreate(&g_uwboadTaskID, &stTaskInitParam);
+    if (uwRet != LOS_OK)
+    {
+        return ;
+    }
+    return ;
```

Comment:
Why the space before the semicolons? (Seems this function returned a value earlier and that was deleted, but spacing was not adjusted.)